doc/PLUGINS.md

```diff
@@ -1508,28 +1508,34 @@ type prefix, since c-lightning does not know how to parse the message.
 Because this is a chained hook, the daemon expects the result to be
 `{'result': 'continue'}`. It will fail if something else is returned.
 
-### `onion_message` and `onion_message_blinded`
+### `onion_message`, `onion_message_blinded` and `onion_message_ourpath`
 
 **(WARNING: experimental-offers only)**
 
-These two hooks are almost identical, in that they are called when an
-onion message is received.  The former is only used for unblinded
-messages (where the source knows that it is sending to this node), and
-the latter for blinded messages (where the source doesn't know that
-this node is the destination).  The latter hook will have a
-"blinding_in" field, the former never will.
+These three hooks are almost identical, in that they are called when
+an onion message is received.  The `onion_message` hook is only used
+for obsolete unblinded messages, and can be ignored for modern usage.
```

Comment:
`obsolete` sounds very harsh, especially in the docs, but I can't come up with a better word.

Reply:
It will get removed next version anyway, so that will resolve the wording debate :)